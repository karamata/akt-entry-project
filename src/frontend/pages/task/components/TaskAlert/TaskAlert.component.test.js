import React from 'react';
import renderer from 'react-test-renderer';

import TaskAlertComponent from './TaskAlert.component';

jest.mock('antd/dist/antd.css', () => <div />);

it('Alert success when showed', () => {
  const component = renderer.create(<TaskAlertComponent isError={false} message={'Entity created'} visible={true} />);
  expect(component).toMatchSnapshot();
});

it('Alert failed when showed', () => {
  const component = renderer.create(<TaskAlertComponent isError={true} message={'Title required'} visible={true} />);
  expect(component).toMatchSnapshot();
});
