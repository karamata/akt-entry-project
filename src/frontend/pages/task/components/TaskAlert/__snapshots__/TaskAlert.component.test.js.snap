// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert failed when showed 1`] = `
<div
  className="ant-alert ant-alert-error ant-alert-no-icon"
  data-show={true}
>
  <span
    className="ant-alert-message"
  >
    <span>
      <b>
        Error!
      </b>
       
      Title required
    </span>
  </span>
  <span
    className="ant-alert-description"
  />
</div>
`;

exports[`Alert success when showed 1`] = `
<div
  className="ant-alert ant-alert-success ant-alert-no-icon"
  data-show={true}
>
  <span
    className="ant-alert-message"
  >
    <span>
      <b>
        Success!
      </b>
       
      Entity created
    </span>
  </span>
  <span
    className="ant-alert-description"
  />
</div>
`;
